src/lib/remote-place-cache.ts

```diff
@@ -47,23 +70,35 @@ export default class RemotePlaceCache {
   public static add(place: Place, chtApi: ChtApi): void {
     const { domain } = chtApi.chtSession.authInfo;
     const placeType = place.type.name;
-
-    const places = this.cache[domain]?.[placeType];
-    // if there is no cache existing, discard the value
-    // it will be fetched if needed when the cache is built
-    if (places) {
+    const cacheKey = this.getCacheKey(domain, placeType);
```

Comment:
lets say there is a place `x` of type `type` on the instance. 

scenario 1
1. the cache is empty
2. run get('type') 
you'll get x

but scenario 2
1. the cache is empty
2. add(y) of `type`
3. get('type') 

now you won't see x.  i think this is a bug, no?